src/lib/services/event-service.ts

```diff
@@ -28,6 +28,10 @@ export default class EventService {
             (e: IEvent) => e.type !== FEATURE_METADATA_UPDATED,
         );
     }
+
+    async getEventsForProject(project: string): Promise<IEvent[]> {
+        return this.eventStore.getEventsFilterByProject(project);
```

Comment:
**side note:*'
All stores now support `getAll(query: QT)` so we should in theory be able to combine these into a single getAll-call to the store.

Reply:
ooh, good point. Let's make a separate task for that :)